movieApp.js

```diff
@@ -107,18 +101,149 @@ for (let movie of movies) {
   }
   createCommentSection();
 
-  readMoreBtn.addEventListener("click", function onclick() {
-    // location.href = "_blank";
-    // Create element to show details about the movie
-
-    /* Toggle between adding and removing the "active" class,
-     to highlight the button that controls the panel */
-    this.classList.toggle("active");
-    /* Toggle between hiding and showing the active panel */
-    if (movieDetails.style.display === "block") {
-      movieDetails.style.display = "none";
-    } else {
-      movieDetails.style.display = "block";
+  function toggle() {
```

Comment:
It's always a good idea to make sure function names (also variable names etc) are specific and descriptive. Here, one could ask "toggle what?" :)

Reply:
yes, I will specify the name of function